backend/src/defence.ts

```diff
@@ -144,6 +186,22 @@ function generateRandomString(string_length: number) {
   return random_string;
 }
 
+// check message for any words in the filter list
+function detectFilterList(message: string, filterList: string) {
+  const detectedPhrases = [];
+  const cleanedMessage = message.replace(/[^a-zA-Z ]/g, "").toLowerCase();
```

Comment:
I think it would be good to filter on any string, including ones containing symbols and numbers if possible